packages/loot-core/src/server/budget/app.ts

```
@@ -153,6 +165,390 @@
+  const categoryGroups = await getCategoryGroups();
+  const projections = {};
+
+  // Load template if using manual projection
+  let template = null;
+  if (projectionType === 'manual' && templateId) {
+    const templateData = await db.first(
+      'SELECT * FROM future_templates WHERE id = ?',
+      [templateId],
+    );
+    if (templateData) {
+      template = JSON.parse((templateData as any).template);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)